interface-definitions/system-config-mgmt.xml.in

```diff
@@ -21,7 +21,7 @@
                     <description>Uniform Resource Identifier</description>
                   </valueHelp>
                   <constraint>
-                    <validator name="url --file-transport"/>
```

Comment:
Multiple validators work as logical OR, so you don't need to replace the whole thing. You can just add a regex validator for git specifically.

Reply:
Good to know, fixed in new commit